config/opal_config_asm.m4

```diff
@@ -1086,6 +1087,7 @@ dnl
 dnl #################################################################
 AC_DEFUN([OPAL_CONFIG_ASM],[
     AC_REQUIRE([OPAL_SETUP_CC])
+    AC_REQUIRE([OPAL_SETUP_CXX])
```

Comment:
Does this Require both a C and C++ compiler?

Reply:
No - I can put this elsewhere. I just dumped it here because that is where the CC was setup. So I just put it here for consistency.

Reply:
Looks like `CXX=/fake/compiler` before this, rather than being unset.  That's not caused by this PR, just a strange convention.  So this seems like a fine place to call `AC_REQUIRE([OPAL_SETUP_CXX])`